Issue #370 - remove lifecycle status from readme

bailliem opened the pull request:
**What changes are proposed in this pull request?**

Resolves issue #370 

Removes lifecycle badge and text from Readme.Rmd. 
The remaining badges have been placed under the installation section. 

**Which files are involved in this pull request and what changes were made?**

Readme.Rmd

**Did you include unit tests for the proposed change/bug fix (https://testthat.r-lib.org/)?**

N/A

**If there is an GitHub issue associated with this pull request, please provide link.**

closes #370
 
--------------------------------------------------------------------------------

Checklist for PR reviewer

- [x] PR branch has pulled the most recent updates from main branch. Ensure the pull request branch and your local version match and both have the latest updates from the main branch.
- [x] If a new function was added, function should be included in `_pkgdown.yml`
- [x] If a bug was fixed, a unit test was added for the bug check
- [x] Run `pkgdown::build_site()`. Check the R console for errors, and review the rendered website.
- [x] Code coverage is suitable for any new functions/features. Review coverage with `withr::with_envvar(new = c("NOT_CRAN" = "true"), covr::report())`. Before you run, begin a fresh R session without any packages loaded. 
- [x] R CMD Check runs without errors, warnings, and notes
- [x] `usethis::use_spell_check()` runs with no spelling errors in documentation
- [x] Has `NEWS.md` been updated with the changes from this pull request under the heading indicating the latest version. If there is an issue associated with the pull request, reference it in parentheses at the end update (see `NEWS.md` for examples).
- [x] Has the version number been incremented using `usethis::use_version(which = "dev")` 
- [x] Approve Pull Request
- [x] Merge the PR. Please use "Squash and merge".

ddsjoberg commented:
The standard placement of the badges is directly below the first header. Unless there is a specific reason to move them in this case, I think we should just keep them in the default location. (This is nit-picky and doesn't really matter!)

Here's the output from `usethis::create_tidy_package()`.

````
---
output: github_document
---

<!-- README.md is generated from README.Rmd. Please edit that file -->

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  fig.path = "man/figures/README-",
  out.width = "100%"
)
```

# testing.pkg

<!-- badges: start -->
[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
[![CRAN status](https://www.r-pkg.org/badges/version/testing.pkg)](https://CRAN.R-project.org/package=testing.pkg)
<!-- badges: end -->

The goal of testing.pkg is to ...

## Installation

You can install the development version of testing.pkg like so:

``` r
# FILL THIS IN! HOW CAN PEOPLE INSTALL YOUR DEV PACKAGE?
```

## Example

This is a basic example which shows you how to solve a common problem:
````

bailliem commented:
@ddsjoberg  - thanks, I agree. Makes sense to be consistent - I was looking for where to place it after we removed the lifecycle section. Now moved to under the fist section.
